### PR TITLE
BE-140 Reset pass endpoints

### PR DESCRIPTION
**Pull request by gaaral**

### CODE REVIEW

File: app/Http/Controllers/Api/PasswordResetController.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+        $user = User::where('email', $passwordReset->email)->first();
+
+        throw_unless($user,
+            BadRequestHttpException::class,
+            "We can't find user with that e-mail address.");
+
+        $user->password = $request->password;
+        $user->save();
+//        $passwordReset->delete();
```
Comment (gaaral) [inline app/Http/Controllers/Api/PasswordResetController.php]: uncomment it

File: app/Http/Controllers/Api/PasswordResetController.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    /**
+     * Find reset password token
+     *
+     * @param $token
+     * @return \Illuminate\Http\JsonResponse
+     * @throws \Throwable
+     */
+    public function find($token)
+    {
+        $passwordReset = PasswordReset::where('token', $token)
```
Comment (lexod) [inline app/Http/Controllers/Api/PasswordResetController.php]: Add a binding to RouteServiceProvider

File: app/Http/Controllers/Api/PasswordResetController.php (hunk 1 above)
Comment (gaaral) [reply app/Http/Controllers/Api/PasswordResetController.php]: done

File: app/Http/Controllers/Api/PasswordResetController.php (hunk 2 above)
Comment (gaaral) [reply app/Http/Controllers/Api/PasswordResetController.php]: done

**Pull request merged**